package.json

```diff
@@ -35,12 +35,13 @@
     "bcryptjs": "^2.4.3",
     "body-parser": "^1.19.0",
     "bootstrap": "^4.5.2",
-    "darkreader": "^4.9.16",
     "chart.js": "^2.9.3",
+    "darkreader": "^4.9.16",
```

Comment:
lol why'd it swap. weird

Reply:
I think it could be when I rebase and accept the incoming changes